webui/ec2/EC2.py

```diff
@@ -23,78 +23,120 @@ def _user_add_keys(user):
             k.create_date = key.get('CreateDate')
             k.key_id = key.get('AccessKeyId')
             k.status = key.get('Status').lower()
-            user.keys.append(k);
+            user.keys.append(k)
     return user
 
 
-def get_users(name = None):
+def get_users(name=None):
     '''
-    See https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html
-    for more details on boto iam api.
+    See for more details on boto iam api.
+        https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html
     '''
     iam = boto3.client('iam', aws_access_key_id=provider_conf.EC2['key'],
                        aws_secret_access_key=provider_conf.EC2['secret'])
     users = []
     if (name):
         try:
-            response = iam.get_user(UserName = name)
+            response = iam.get_user(UserName=name)
             user = response.get('User')
-            u = _ec2user_to_user(user);
-            _user_add_keys(u);
-            users.append(u);
+            u = _ec2user_to_user(user)
+            _user_add_keys(u)
+            users.append(u)
         except ClientError as ex:
-            print (ex);
+            print(ex)
```

Comment:
maybe like future improvement , but we need something bigger then `print` for logging , like some real logging framework

Reply:
+1 as a hint https://docs.python.org/2/howto/logging.html#a-simple-example

Reply:
not needed now, just a suggestion for future improvement :)